File: program/Livro.java
package program;

public class Livro 
{
    private String isbn;
    private String nome;

    public Livro(String isbn, String nome)
    {
        this.isbn = isbn;
        this.nome = nome;
    }

    public String getISBN()
    {
        return this.isbn;
    }

    public String getNome()
    {
        return this.nome;
    }
}

File: program/livro.py
class Livro:

    def __init__(self, isbn, nome, autor):
        
        self.isbn  = isbn
        self.nome  = nome
        self.autor = autor

    def __eq__(self, isbn):
        
        return self.isbn == isbn
    
    def exportar(self, caminho):

        with open(caminho + '/' + self.nome + '.txt', 'w') as file:
            
            file.write(f'{self.nome} - {self.autor.nome}')